completion/git-machete.completion.zsh

```diff
@@ -34,6 +34,12 @@ _git-machete() {
             '(-H --sync-github-prs)'{-H,--sync-github-prs}'[Annotate with GitHub PR numbers and authors where applicable]' \
           && ret=0
           ;;
+        (clean)
+          _arguments \
+            '(--checkout-my-github-prs)'--checkout-my-github-prs'[Checkout your open PRs into local branches]' \
+            '(-y --yes)'{-y,--yes}'[Do not ask for confirmation]' \
```

Comment:
```suggestion
            '(-y --yes)'{-y,--yes}'[Do not ask for confirmation when deleting branches]' \
```
Actually, I'd specify whether it's untracked, unmanaged or both

Reply:
👍🏼